### DISCUSSION TITLE
keyball61で、トラックボールが動かない

### DISCUSSION THREAD

**Discussion by degu-blank** [Yowkees/keyball#249] [Q&A] [answered]
一通り組み終わったのですが、
トラックボールが動かず困っています。
確認すべきところがありましたら教えてください。

- keyball61で、右手側にトラックボールをつけています。
- ProMicroとトラックボール基盤のはんだは再度熱を当てましたが変わらずでした。
  - どこのはんだを見直せばよいかはわかっていません
 - トラックボールは一度も反応したことがありません

画像
| 1 | ２ | 3 | 4 |
| --- | --- | --- | --- |
| ![S__58064899](https://github.com/Yowkees/keyball/assets/136141560/31322dea-8920-4f9a-a80c-2bdd760556c3) | ![S__58064901](https://github.com/Yowkees/keyball/assets/136141560/feb02872-3f70-4bd1-8f18-96fc73bffaa4) | ![S__58064902](https://github.com/Yowkees/keyball/assets/136141560/3da16a3f-a43c-4f2d-b493-a7de4c77a943) | ![S__58064903](https://github.com/Yowkees/keyball/assets/136141560/0db3f6ce-33fb-4062-8dff-cd4038bbc26e) |

お手数おかけしますがご確認よろしくお願いいたします。
必要な情報や画像がありましたら教えてください。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。
17番のキーはスイッチの足が折れているように写真では見えますがいかがでしょうか。

また、トラックボールについては、下記写真の同色どうしが導通しているかテスター等でご確認いただくことは可能でしょうか。
![KB61_TBfix](https://github.com/Yowkees/keyball/assets/84209772/e01091b1-fdb6-4c99-b71f-02e258e041e0)

導通しているようでしたら、センサーの故障の可能性があります。
その場合、基板単体での販売等可能ですので、ご連絡いただければ幸いです。

以上、お手数ですがご確認お願いいたします。

**Reply by degu-blank** [reply]
ありがとうございます。
おっしゃるとおり、キースイッチが折れておりました。スイッチを変えたところ治りました。

導通チェックしてみましたが、すべて問題なさそうでした。

念のための確認ですが、ファームウェアを入れられた時点でトラックボールは動きますか？
@Yowkees

**Comment by Yowkees** [comment]
ファームウェアを入れた時点でトラックボールが動きます。
ProMicro裏にて、色を付けた端子が他の端子とショートしてしまっていないかもご確認おねがいします。

**Reply by degu-blank** [reply]
迅速な回答ありがとうございます。
写真の部分であれば、ショートしていないようにみえます。
@Yowkees 

| ProMicro |
| --- |
| ![S__58064921](https://github.com/Yowkees/keyball/assets/136141560/4078ed0f-07f7-4b7a-899f-b0f095c16a05) |

**Reply by Yowkees** [reply]
ProMicro本体は大丈夫そうですので、ミドル基板上で信号どうしのショートがないか確認いただければ幸いです。
